cassandra_backup_service.py

```diff
@@ -2183,7 +2215,9 @@ def restore(self, columnfamily, nodes, restore_time=None, restore_dir=None, host
                                      help='Optionally provide username to use when connecting to CQLSH')
             repo_parser.add_argument('--cqlsh-pass', dest='cqlsh_pass', required=False,
                                      help='Optionally provide password to use when connecting to CQLSH')
-
+            # Manifest options
+            repo_parser.add_argument('--retention-days', dest='retention_days', required=False, default=30,
```

Comment:
Can we remove the 30 day default here and by default retain indefinitely.

Reply:
Removed default: https://github.com/onzra/cassandra_backup_service/pull/43/commits/cd0d5b7cfc6ab0f574ad3e781e83734f96719941